docs/release.md

```diff
@@ -30,20 +30,22 @@ git remote add upstream https://github.com/openshift/backplane-cli.git
 
 ### Cutting a new release
 
-Create a tag on the latest master.
+Create a tag on the latest main.
 
 ```bash
 git fetch upstream
-git checkout upstream/master
+git checkout upstream/main
 git tag -a ${VERSION} -m "release ${VERSION}"
 git push upstream $VERSION
 ```
 
+**Note:** We follow [semver](https://semver.org/) for versioning.
```

Comment:
probably worth mentioning that releases should start with a `v` for consistency. I think that's not mandatory in semver per se.